api/v1beta1/integrationtestscenario_suite_test.go

```diff
@@ -22,13 +22,14 @@ import (
 	"fmt"
 	"net"
 	"path/filepath"
-	"sigs.k8s.io/controller-runtime/pkg/metrics/server"
-	crwebhook "sigs.k8s.io/controller-runtime/pkg/webhook"
 	"testing"
 	"time"
 
+	"sigs.k8s.io/controller-runtime/pkg/metrics/server"
```

Comment:
IDE is stubborn about this change, doesn't want it on original place.

Reply:
because you removed line on 108; don't remove it and IDE will not format the file. Just checkout version from main and don't commit it.